### DISCUSSION TITLE
CRON job does not run every day

### DISCUSSION THREAD

**Discussion by by225** [vercel/community#5123] [Help]
### Summary

I have one CRON job which should run every day at the same time. Unfortunately, it does not seem to run every day. It will often skip a day. I get notified by email when the CRON job runs. Whenever I run the CRON job manually, it works exactly as designed. This seems like a bug in the Vercel CRON job timer.

### Steps to Reproduce

Just wait for the next time the CRON job should run and see if it does. The CRON job did not run last night.

Side note: the CRON job always runs later than the time specified in settings, but this is not as critical for me.

**Comment by amyegan** [comment]
Hi @by225. I know that cron jobs on the Hobby plan aren't guaranteed to run at exactly the scheduled time. But a job should run within an hour of it's scheduled time, so I wouldn't expect it to be skipped entirely. Are you able to see any error info logged at the times the job seems to be skipped?

If you let me know the name of the project, or a preview deployment link, I can use that to find the project and get more info about what is going wrong.

**Reply by by225** [reply]
Thanks @amyegan for the reply. I will try checking the logs at the time the CRON job normally runs. My project is by225s-projects and the app is update-brevo.

**Reply by by225** [reply]
Hello @amyegan. The CRON job did not run on 12/7/2023. I was not available during the CRON time to check the logs. But, the CRON job has run perfectly every day since then. Hopefully, this will continue to be the case. Thank you!
